lib/index.ts

```diff
@@ -522,6 +512,15 @@ export default class Backburner {
     this._ensureInstance();
   }
 
+  private handleError(e) {
+    let onError = getOnError(this.options);
+    if (onError) {
+      onError(e);
+    } else {
+      throw e;
```

Comment:
the downside of this is for debuggers + caught on exception stuff.

Before it was:

```js
try {
  invoke(); // debugger will pause at the actual source
} finally {

}
```

Now it is


```js
try {
  invoke();
} catch (e) {
  throw e; // user will end up here, which is lame
} finally {

}
```